### PR TITLE
Fix various instances of d43 links

### PR DESCRIPTION
**Pull request by 16dprice**
This PR addresses various places where the code is linking to door43 sites. These need to be replaced with links pointing to WACS. Several bits of regex are updated along with some other links.

### CODE REVIEW

**Comment by 16dprice** [comment]
This PR is currently marked as a draft because I would like some other eyes to look over this before I'm confident that I've accurately addressed all possible use cases of the code I changed. There are several different links that could be passed into the regex I've written or edited. The tests run and pass. I'd like to make sure I don't need any other tests, I didn't miss any edge cases, etc.

File: tests/client_tests/resources/test_cases/ta_fix_links_tests.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
```
Comment (PurpleGuitar) [inline tests/client_tests/resources/test_cases/ta_fix_links_tests.json:1]: I love this!  Excellent use of data-driven test design.

**Review by PurpleGuitar — APPROVED** [review]
